modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -386,14 +386,16 @@ tinymce.init({
 [NOTE]
 More details about this component and the possible values that can be used within the new `fontsizeinput` input field can be found in `xref:available-toolbar-buttons.adoc#the-core-toolbar-buttons[fontsizeinput]`.
 
-=== Added `skip_focus` check to the ToggleToolbarDrawer command
+=== Added `skipFocus` check to the ToggleToolbarDrawer command
 //#TINY-9337
 
-In previous versions of {productname}, the *ToggleToolbarDrawer* command contained an incorrect focus call. As a consequence, this meant that when the *ToggleToolbarDrawer* command option `skip_focus: true` was set the toolbar drawer would take focus when it was not meant to.
+In previous versions of {productname}, the *ToggleToolbarDrawer* command contained an incorrect focus call. As a consequence, this meant that when the *ToggleToolbarDrawer* command option `skipFocus: true` was set the toolbar drawer would take focus when it was not meant to.
 
-In {productname} 6.4.1, an improvement was made to the focusing code which now checks for `skip_focus`.
+In {productname} 6.4.1, an improvement was made to the focusing code which now checks for `skipFocus`.
 
-As a result of the fix, when using the *ToggleToolbarDrawer* command, the toolbar now avoids stealing focus when `skip_focus: true` is set.
+As a result of the fix, when using the *ToggleToolbarDrawer* command, the toolbar is now not focused when `skipFocus: true` is set.
+
+NOTE: As of {productname} 6.8.3, when using `+skipFocus: true+`, the *ToggleToolbarDrawer* command does not toggle the toolbar drawer when the `+toolbar_mode+` option is set to `+'sliding'+`.
```

Comment:
It seems the issue happened from 6.4.1.
However, I recommend removing (or moving to Known Issues) this Note, since it is weird to put in Release Notes the new option which can not be used since it breaks the command functionality.

Reply:
@vpyshnenko I have moved the note to `known issues` in the 6.4.1 release notes.